qiskit/providers/models/backendconfiguration.py

```diff
@@ -634,9 +641,6 @@ def to_dict(self):
         if self.rep_times:
             out_dict['rep_times'] = [_rt * 1e6 for _rt in self.rep_times]
 
-        if self.rep_delays:
```

Comment:
There needs to be code to undo the scaling for rep_delay (both default and ranges).

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/4830/commits/e46d7a001b29cb978a7fe46db96c155c36d9e2ea.